### PR TITLE
Cleanup repair test

### PR DESCRIPTION
**Pull request by ptnapoleon**
@mambocab to review

### CODE REVIEW

**Comment by ptnapoleon** [comment]
Tests all pass on 3.0, after these non-logic changes

File: repair_test.py [hunk 1]
```diff
@@ -58,12 +61,15 @@ def no_anticompaction_after_dclocal_repair_test(self):
...
+        """
         cluster = self.cluster
         debug("Starting cluster..")
         cluster.populate([2, 2]).start(wait_for_binary_proto=True)
-        [node1_1, node2_1, node1_2, node2_2] = cluster.nodelist()
-        node1_1.stress(stress_options=['write', 'n=50000', 'cl=ONE', '-schema', 'replication(factor=2)'])
-        node1_1.nodetool("repair -hosts 127.0.0.1,127.0.0.2,127.0.0.3,127.0.0.4 keyspace1 standard1")
+        node1_1, node2_1, node1_2, node2_2 = cluster.nodelist()
+        node1_1.stress(stress_options=['write', 'n=50K', 'cl=ONE', '-schema', 'replication(factor=2)'])
+        node1_1.nodetool("repair -hosts 127.0.0.1,127.0.0.2,12  7.0.0.3,127.0.0.4 keyspace1 standard1")
```
Comment (mambocab) [inline repair_test.py]: Intentional? and if the tests pass after this change, what is going on?

**Comment by mambocab** [comment]
+1 other than my comment. Maybe we need assertions to make sure that repair actually happens.

File: repair_test.py (hunk 1 above)
Comment (ptnapoleon) [inline repair_test.py]: Definitely not intentional. Also, scary that test passes

**Comment by ptnapoleon** [comment]
I assume the 
`self.assertTrue(node.grep_log("Not a global repair"))` makes sure the repair happens

**Comment by mambocab** [comment]
The merge is fine, but I'm still concerned. Why didn't nodetool repair fail on what I assume are malformed arguments? Maybe that assertion doesn't do what we think it does.

**Comment by ptnapoleon** [comment]
After rerunning tests, it does fail with that mistake. So I must have accidentally included that space after checking the tests, but before committing.

**Pull request merged**